app.js

```diff
@@ -0,0 +1,37 @@
+const express  = require('express');
+const bodyParser = require('body-parser');
+//const cors = require('cors');
+const routes = express.Router();
+var jwt = require('jsonwebtoken');
+var bcrypt = require('bcrypt');
+const path= require('path')
+const news = require('./src/routes/news');
+const {signin,signup}=require('./src/controllers/authController')
+
+require("dotenv").config();
+
+const app = express();
+//app.use(cors());
+app.use(routes);
+routes.use(bodyParser.urlencoded({ extended: false }));
+routes.use(bodyParser.json());
+
+const PORT = 3000;
+
+
+routes.get('/', (req, res)=>{
+  res.status(200).send("Welcome to the News App");
+});
+
+routes.post('/register',signup)
```

Comment:
all the auth routes could have been registered together. 
For example -: /auth/register, /auth/signin